data_structures/src/error.rs

```diff
@@ -268,6 +268,10 @@ pub enum BlockError {
         fees_value: u64,
         reward_value: u64,
     },
+    #[fail(display = "Different PublicKeyHashes found in the Mint transaction")]
+    DifferentPkhsInMint,
+    #[fail(display = "MintTransaction is split in too small sizes")]
```

Comment:
This error may be hard to understand, but no idea how to improve it.

```suggestion
    #[fail(display = "MintTransaction output is smaller than the collateral minimum")]
```